helm-chart/templates/cronjobs/mongobackup.yaml

```diff
@@ -0,0 +1,256 @@
+{{- if eq .Values.deployEnv "prod" }}
```

Comment:
I am not sure that this or the nationbuilder stuff should be included in the public char